public/app/containers/AlertRuleList/AlertRuleList.tsx

```diff
@@ -173,3 +174,5 @@ export class AlertRuleItem extends React.Component<AlertRuleItemProps, any> {
     );
   }
 }
+
+export default hot(module)(AlertRuleList);
```

Comment:
Having to wrap all containers like this seems a bit intrusive, and repetitive. can’t this be done centrally, for example in react container (the angular component that loads the page react component)